src/main/java/edu/wpi/grip/ui/pipeline/AddSourceView.java

```diff
@@ -52,7 +52,7 @@ public AddSourceView(EventBus eventBus) {
                     eventBus.post(new SourceAddedEvent(new ImageFileSource(eventBus, file)));
                 } catch (IOException e) {
                     e.printStackTrace();
```

Comment:
I would argue that fixing the double or triple stack trace printed would definitely "improve unexpected throwable handling"